src/components/molecules/EarthEditor/PropertyPane/PropertyField/ColorField/index.tsx

```diff
@@ -278,20 +278,20 @@ const layerStyle = css`
   height: 100%;
 `;
 
-const check = ` 
+const check = (props: any) => ` 
```

Comment:
Can the type here be string? Or no good?

Reply:
No, it's not a string.
It contains many properties, including the theme

Reply:
Ya, so currently you are passing ALL props into this check, so I understand why you say that. But you are only using `props.theme.toggleButton.bg` in this check and no other prop or theme entry. Because of that I think it would be cleaner, clearer and type-safed better if you passed in like:
```
background-image: ${({theme})=> check(theme.toggleButton.bg)}, ${props => check(theme.toggleButton.bg)};
```
Then check's property can be typed as a `string` like:
```
const check = (color: string) => ` 
  linear-gradient(
    45deg, 
    ${color} 25%, 
    transparent 25%, 
    transparent 75%, 
    ${color} 25%, 
    ${color} 
  ) 
`;
```